docker-plain/files/chart/values.yaml

```diff
@@ -0,0 +1,127 @@
+
+## The number of replicas to deploy.
+## For high availability use more than 1 replica
+replicaCount: 1
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# NOTE: By default image Values are injected from CICD Jenkins pipeline, values defined here are used when not being on CICD Jenkins pipeline context.
+# Default values here are provided in case one needs to use the chart without CICD (i.e.: testing the chart).
+image:
+  # registry: "public.ecr.aws"
+  # path: "nginx"
+  # name: "nginx-unprivileged"
+  # tag: "alpine-slim"
+  # see https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+  pullPolicy: IfNotPresent
+
+## Prefer using ingress over openshift routes
+ingress:
+  enabled: false
+  className: 'openshift-default'
+  # router: external
+  annotations:
+    ## adjust openshift timeouts (default is 60s)
+    haproxy.router.openshift.io/timeout: 300s
+    haproxy.router.openshift.io/timeout-tunnel: 300s
+    ## e.g. add cert-manager support by annotating the ingress https://cert-manager.io/docs/usage/ingress/
+    ## ask in your company for good defaults
+
+  hosts: [] # When defining a host you must define also a path
+  # - host: yourapp.yourdomain.com
+  #   paths:
+  #     - path: /
+  #       pathType: Prefix
+  tls: [] # If `tls` is left empty then the default OpenShift TLS config will be loaded (i.e.: TLS edge termination with HTTP redirect to HTTPS)
+  # - secretName: chart-example-tls
+  #   hosts:
+  #     - yourapp.yourdomain.com
+
+service:
+  enabled: true
+  port: 8080
+  type: ClusterIP
+
+# There are two types of strategy: `Recreate` and `RollingUpdate`
+# Please refer to https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+deploymentStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 0%
+    maxSurge: 50%
+
+probes:
+  livenessProbe:
+    failureThreshold: 3
+    httpGet:
+      path: "/health"
+      port: 8080
+      scheme: HTTP
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 3
+
+  readinessProbe:
+    failureThreshold: 1
+    httpGet:
+      path: "/health"
+      port: 8080
+      scheme: HTTP
+    initialDelaySeconds: 3
+    periodSeconds: 5
+    successThreshold: 1
+    timeoutSeconds: 3
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  # name: "default"
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 32Mi
+  requests:
+    cpu: 10m
+    memory: 16Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 3
```

Comment:
The Release Manager doesn't support multiple replicas yet. Not sure if these values should even be configurable, or at least maxReplicas should be 1 by default, with a comment saying that higher values aren't supported.

Reply:
@jafarre-bi I don't know about this limitation, there's nothing in opendevstack documentation about. Please provide the source

Reply:
HelmDeploymentStrategy, line 198:
// TODO: Once the orchestration pipeline can deal with multiple replicas,
// update this to store multiple pod artifacts.
Even if the code could deal with it and the documentation and traceability would be correctly preserved, it's still not officially supported. We would need to validate the functionality, document it and create specific automated tests to prevent regressions.

Reply:
As agreed, I will see if we are on time to add a note stating that multiple replicas are currently unsupported to the official documentation. We will add a comment to this values file referencing that to warn the users to use this at their own risk.